### ISSUE TITLE
Support for Pub spec

### ISSUE THREAD

**Issue by danielgomezrico** [pivotal/LicenseFinder#804]
This is the package manager used in flutter and dart

https://pub.dev/

**Comment by cguentherTUChemnitz** [comment]
> This is the package manager used in flutter and dart
> 
> https://pub.dev/

Yeah that would be super nice to have. Currently we are able to get all licence information for all dependencies in our development stack except for flutter and dart. Having this in place would simplify the situation a lot for us.

**Comment by etiennecadicidean** [comment]
Hey there, 
should be available when this PR gets approved 
https://github.com/pivotal/LicenseFinder/pull/875

for some reasons, the CI build fails for some reasons that are not linked to the changes made... I'm waiting for the repo and @xtreme-shane-lattanzio  to proceed

**Comment by xtreme-shane-lattanzio** [comment]
Hey @etiennecadicidean ! We are having some CI issues right now. Once we fix them, I will rebase the PR to make sure its all good and merge in if its good. Thanks for making this!
